docker/Dockerfile

```diff
@@ -8,6 +8,14 @@ RUN apt-get install -y nodejs postgresql postgresql-contrib
 # To allow tests to be run with the test database
 ENV DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL true
 
-COPY ./ /srv/app/datagovuk_publish
+ENV DATAGOVUK_HOME /srv/app/datagovuk_publish
+
+# Create datagovuk user
+RUN useradd -r -u 900 -m -c "datagovuk account" -d $DATAGOVUK_HOME -s /bin/false datagovuk
+
+COPY ./ $DATAGOVUK_HOME
 
 RUN gem install bundler && bundle install
+
+RUN chown -R datagovuk:datagovuk $DATAGOVUK_HOME
```

Comment:
The app shouldn't need to write to the container image, so it shouldn't need to be the owner of any files in there. Do you know if this is really needed? If it is, then let's call it out as a TODO to fix it later at least.

Reply:
I haven't looked into it in too much details but things seem to work from a fresh startup with the new Dockerfile, I'll add a TODO for future fix